Rollbar/RollbarNotifier.m

```diff
@@ -356,7 +378,12 @@ - (NSDictionary*)buildPayloadBodyWithMessage:(NSString*)message extra:(NSDiction
 }
 
 - (NSDictionary*)buildPayloadBodyWithException:(NSException*)exception {
-    NSDictionary *exceptionInfo = @{@"class": NSStringFromClass([exception class]), @"message": exception.reason, @"description": exception.description};
+
```

Comment:
What is going on here? Why change this?

Reply:
making sure that the exception's message and description are mutable strings since they are potential candidates for truncation in general...